web/client/components/map/cesium/Map.jsx

```diff
@@ -327,7 +329,7 @@ class CesiumMap extends React.Component {
         }) : null;
         const ErrorPanel = this.props.errorPanel;
         return (
-            <div id={this.props.id}>
+            <div id={this.props.id} data-sidebar={this.props.sidebar ? true : null}>
```

Comment:
Map components and plugins should not be aware of the sidebar plugin.
In my opinion we could keep the 40px margin always without detect if the burger menu is active or not.
So please remove this data and the related prop.